cli/flox/doc/flox-install.md

```diff
@@ -48,11 +56,10 @@ ID is set to that attribute.
 For pkg-paths that consist of multiple attributes (e.g. `python310Packages.pip`)
 the install ID is set to the last attribute in the pkg-path (e.g. `pip`).
 
-You may also specify packages to be installed via
-[`flox-edit(1)`](./flox-edit.md),
-which allows specifying a variety of options for package installation.
-See [`manfifest-toml(1)`](./manifest.toml.md) for more details on the available
-options.
+If a flake output attribute is specified instead of a pkg-path, the install ID
+is inferred from the attribute path specified,
+or if no attribute path is provided, the install ID is inferred from the flake
+reference.
```

Comment:
Suggestion: I think we need to mention somewhere that this is explicitly an escape hatch meant for Nix power users, and most people will not need this feature. Otherwise it's not clear to a user when this feature is necessary or why they should care about it.

Reply:
Called it an "advanced feature," wdyt?